crx_template/content.js

```diff
@@ -80,8 +80,8 @@
     document.documentElement.className += ' FB_FW_ext ${PLUGIN_NAME}';
 
     injectScript("FireBreathPromise.js");
-    injectScript("wyrmhole.js");
     injectScript("firewyrm.js");
-
+    injectScript("wyrmhole.js");
```

Comment:
If we're going to do this this way, I'd prefer to follow the pattern on both backend and frontend; perhaps shared code in wyrmhole.js, page content code in wyrmhole-content.js, and background code in wyrmhole-background.js

As it is we're injecting a lot of javascript code into every page load that isn't used or needed.